lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -257,7 +257,7 @@ def set_segments(self, segments):
         """
         Set 3D segments.
         """
-        self._segments3d = np.asanyarray(segments)
+        self._segments3d = segments
```

Comment:
This one seems worrying both because we are losing an assurance that something is numpy-like and that we are getting segments that are ragged!

Reply:
:sheep: never mind, I am read the commit message and now understand why this is correct and I am wrong.